x/group/module/abci.go

```diff
@@ -0,0 +1,33 @@
+package module
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/group"
+	"github.com/cosmos/cosmos-sdk/x/group/keeper"
+)
+
+func EndBlocker(ctx sdk.Context, k keeper.Keeper) {
+	updateTallyOfVPEndProposals(ctx, k)
+}
+
+func updateTallyOfVPEndProposals(ctx sdk.Context, k keeper.Keeper) error {
+	k.IterateVPEndProposals(ctx, sdk.FormatTimeBytes(ctx.BlockTime()), func(proposal group.Proposal) (stop bool) {
```

Comment:
What do people think of emitting an event with the tally reslut and the updated proposal status/result on EndBlocker?